### ISSUE TITLE
Dependency error with SvelteKit v1.0.0-next.137

### ISSUE THREAD

**Issue by nstuyvesant** [bestguy/sveltestrap#344]
Getting this error twice when I run the dev server `npm run dev -- --host --open`...
> ENOENT: no such file or directory, open '/Users/nates/dev/shy-svelte/node_modules/sveltestrap/src/node_modules/sveltestrap/src/InlineContainer.svelte'

I am not referencing InlineContainer.svelte in any of my code but I am using Modal and Offcanvas which both reference it.

Using sveltestrap@5.5.0, SvelteKit 1.0.0-next.137, Svelte 3.40.2, vite 2.4.3, node 16.5.0

Got the same error creating a pristine project putting Offcanvas in the boilerplate index.svelte...

- `npm init svelte@next my-app' (setup the demo app with TypeScript, eslint and prettier)
- `cd my-app`
- `npm install --saveDev sveltestrap`
- Added code to /src/routes/index.svelte script tag...
```typescript
import { Offcanvas } from 'sveltestrap/src'
let isOpen = true
const toggle = () => {
    isOpen = !isOpen
  }
```
Added this to the section tag...
```html
<Offcanvas {isOpen} {toggle} placement="end" header="This is a test">
Test
</Offcanvas>
```

**Comment by bestguy** [comment]
Hmm weird, let me see what's up

**Comment by nstuyvesant** [comment]
Thanks - just updated my initial report with more details.

**Comment by bestguy** [comment]
Hi @nstuyvesant, can you try this prerelease?

`npm install sveltestrap@5.5.1-0` 

Am assuming should fix but please confirm and I can do a real release.

**Comment by bestguy** [comment]
Actually disregard - trying this locally and still seeing:

`/.../my-app-test/node_modules/sveltestrap/src/node_modules/sveltestrap/src/InlineContainer.svelte`

What's up with this `node_modules/sveltestrap/src/node_modules` stuff?

**Comment by bestguy** [comment]
Hey @nstuyvesant , I think I found the issue - Sveltekit freaks out because InlineContainer.svelte has no <script></script> tag in it.  This is not an issue with other tools so am assuming that's an error?  

I have this PR which works around though: #345 
Works fine for me locally but please try `npm install sveltestrap@5.5.1-3` to confirm.

**Comment by nstuyvesant** [comment]
Thanks for this help! I can confirm sveltestrap@5.5.1-3 solves this issue.

I can also add that if a Svelte component has a <script></script> tag but nothing in it, the same problem occurs.

**Comment by nstuyvesant** [comment]
Is this bug in SvelteKit or Svelte (seems like the former)? Do you want to log the issue on the appropriate Github repo or prefer for me to put it in?

**Comment by nstuyvesant** [comment]
Actually, this issue was submitted already... https://github.com/sveltejs/kit/issues/2001

**Comment by bestguy** [comment]
Cool thanks.  I'll release a new version with this workaround for the time being.

**Comment by bestguy** [comment]
Released in `sveltestrap@5.5.1`, thanks for the heads up 👍

**Comment by nstuyvesant** [comment]
Awesome - thanks for the quick turnaround!
